apps/sms/test/marionette/lib/views/dialer.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
It feels like this view should be somewhere in Dialer folder, close to source it depends on.